src/correlator_bosonic_recursive.jl

```diff
@@ -1,4 +1,5 @@
-using TupleTools: TupleTools
+#using TupleTools: TupleTools
+using Combinatorics
```

Comment:
What operation is being used from `Combinatorics`? Please list it explicitly here, i.e. `using Combinatorics: f` if you are using function `f` in the source code.

Reply:
I am using the function `parity()`, will fix in next commit together with dependency.